.github/ISSUE_TEMPLATE/default.md

```diff
@@ -0,0 +1,10 @@
+## Description
+
+<!--
+Please provide a detailed description of how the issue. 
```

Comment:
"how the issue" sounds like a sentence fragment. What about:
```suggestion
Please provide a detailed description of how to reproduce the issue. 
```

Reply:
Updated 👍